### PR TITLE
[v5] `state.configuration` -> `state._nodes`

### PR DESCRIPTION
**Pull request by davidkpiano**
The `state.configuration` property has been renamed to `state._nodes`.

```diff
- state.configuration
+ state._nodes
```

### CODE REVIEW

File: packages/core/src/State.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ interface MachineSnapshotBase<
   /**
    * The enabled state nodes representative of the state value.
    */
-  configuration: Array<StateNode<TContext, TEvent>>;
+  nodes: Array<StateNode<TContext, TEvent>>;
```
Comment (Andarist) [inline packages/core/src/State.ts:93]: while we are at it I'd propose to rename this to `_nodes`

Comment (davidkpiano) [reply packages/core/src/State.ts:93]: Sure - go for it

File: packages/core/test/typegenTypes.test.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import {
 } from '../src/index.ts';
 import { fromPromise } from '../src/actors/index.ts';
 import { fromCallback } from '../src/actors/index.ts';
-import { createMachine } from '../src/Machine.ts';
+import { createMachine } from '../src/createMachine.ts';
```
Comment (Andarist) [inline packages/core/test/typegenTypes.test.ts:10]: bold move

**Review by Andarist — APPROVED** [review]

File: packages/core/test/typegenTypes.test.ts (hunk 2 above)
Comment (davidkpiano) [reply packages/core/test/typegenTypes.test.ts:10]: Needed to be done

**Pull request merged**